ground/src/main/res/layout/multiple_choice_checkbox_item.xml

```diff
@@ -62,9 +61,10 @@
         android:id="@+id/user_response_text"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:hint="@string/other"
+        android:fontFamily="@font/text_500"
         android:inputType="text"
-        android:text="@{item.otherText}"
+        android:textColor="@color/md_theme_onSurface"
+        android:textSize="16sp"
```

Comment:
I believe you'd want to extend `Widget.MaterialComponents.TextInputEditText.FilledBox` here.

Reply:
Obsolete since the UI has been migrated to compose.